packages/cli/src/commands/build.ts

```diff
@@ -55,3 +41,32 @@ async function copyResource(from: string, to: string) {
     console.error(`${chalk.red('error')} - ${err}`)
   }
 }
+
+async function cleanup() {
```

Comment:
I'm not sure I like this function's name. I think this is doing two different things, right?

1. Cleaning up the `node_modules` from the `tmpDir`
2. Preparing the standalone folder.

So maybe have two different functions for that?

Reply:
In 2 we fix the standalone output dir by removing the `.faststore` from there, so I thought to keep them together, but I'm ok with separating it